Gemfile

```diff
@@ -48,6 +50,9 @@ group :development do
   # Browser simulation
   gem 'capybara'
   gem 'selenium-webdriver'
+
+  # Debugging
+  gem 'pry-rails'
```

Comment:
We already have `byebug`, which is the default for Rails 6. I'm happy with either, but I think we should only use one. What do you think?

Reply:
Sorry I didn't intend to leave this in here, good catch. Have removed.